/**:
  ros__parameters:
    CameraExtrinsicMat_right: [-0.444861602662785,	-0.895597661154733,	-0.00172737097061844,	-0.471096446157682,
-0.392172626025588,	0.196533394169747,	-0.898651910570669,	0.387710893100130,
0.805170035379169,	-0.399098301562773,	-0.438659047344537,	0.794923048789547,
0.,	0.,	0.,	1.]
    CameraMat_right: [495.961006648953,	0.,	310.180126905459,
0.,	526.314316456211,	245.544832187264,
0.,	0.,	1.]
    DistCoeff_right: [0.0857581603185306,	-0.205107118708201, 0.,   0.]
    CameraExtrinsicMat_left: [0.481729890043554,	-0.875951523854791,	-0.0254015923732266,	0.486424571165374,
      -0.387210520349857,	-0.186764175299685,	-0.902877154298979,	0.400899427263234,
      0.786132511710983,	0.444778676063836,	-0.429147530984256,	0.804955977129503,
      0.0,	0.0,	0.0,	1.0]
    CameraMat_left: [500.509183966071,	0.,	327.463855595137,
      0.,	516.454012492216,	240.161668009618,
      0.,	0.,	1.]
    DistCoeff_left: [0.0876473408233649, -0.190713986035280, 0.0,   0.0]

#1006